src/features/fetchEventHandler.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run `wrangler dev src/index.ts` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `wrangler publish src/index.ts --name my-worker` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+/* eslint-disable @typescript-eslint/object-curly-spacing */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+import Logger from '../infras/logger';
+
+type Env = {
+  [key: string]: string;
+  KV: KVNamespace;
+};
+
+export default async function handleRequest(request: Request, env: Env) {
+  const logger = new Logger(env);
+  try {
+    if (request.method === 'POST') {
+      const payload = await request.json();
+      if (payload?.message?.from && !payload.message.from.is_bot) {
+        const kvKey = `conversation-with-${payload.message.from.username}`;
+        const past = await env.KV.get(kvKey, { type: 'json' });
+        const kvValue = { ...past };
+        kvValue[payload.message.date] = kvValue.hasOwnProperty(
+          payload.message.date,
+        )
+          ? [...kvValue[payload.message.date], payload.message.text]
+          : (kvValue[payload.message.date] = [payload.message.text]);
+        await env.KV.put(kvKey, JSON.stringify(kvValue));
+
+        if (!payload.message.from.is_bot) {
```

Comment:
Duplicated `!payload.message.from.is_bot`. The KV was intended to store conversation with bot as well